latex/ledger-spec.tex

```diff
@@ -606,7 +860,8 @@ \subsection{Witnesses}
     \left(
       \begin{array}{r@{~\in~}lr}
         \var{utxo} & \UTxO & \text{UTxO}\\
-        \var{dwstate} & \DWState & \text{delegation witnesess state}\\
+        \var{dstate} & \DState & \text{delegation state}\\
+        \var{pstate} & \PState & \text{pool state}\\
```

Comment:
was this intentional?

Reply:
I am not sure what happened! Not intentional